proto/aep-api/aep/api/field_behavior.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package aep.api;
+
+import "google/protobuf/descriptor.proto";
+import "google/protobuf/duration.proto";
+
+option cc_enable_arenas = true;
+option go_package = "aep.dev/api";
+option java_multiple_files = true;
+option java_outer_classname = "FieldBehaviorProto";
```

Comment:
was this name intentional? It'll conflict with AIP fieldbehavior if we adopt those: https://github.com/googleapis/googleapis/blob/master/google/api/field_behavior.proto#L49.

I'd suggest a different name, like `dev.aep.api.Field`. Or maybe `dev.aep.api.FieldInfo` to match the aip one (see below comment).

Reply:
It was intentional, but I'm open to changing it.  That said would it actually collide, given that the java package is different?  Or do you mean it might "collide" in human brains rather than in the compiler?

I kind of view this as doing more or less the same thing, in spirit, as `google.api.FieldBehavior`, which is why I gave it the same name.  Is it more or less confusing to pick a different name for something that is the same concept?  I don't know!

For reference, here's the generated Java code as the PR currently is:

```java
// Generated by the protocol buffer compiler.  DO NOT EDIT!
// source: aep/api/field_behavior.proto

// Protobuf Java Version: 3.25.1
package dev.aep.api;

public final class FieldBehaviorProto {
  private FieldBehaviorProto() {}
  public static void registerAllExtensions(
      com.google.protobuf.ExtensionRegistryLite registry) {
    registry.add(dev.aep.api.FieldBehaviorProto.minimumLifetime);
  }

  public static void registerAllExtensions(
      com.google.protobuf.ExtensionRegistry registry) {
    registerAllExtensions(
        (com.google.protobuf.ExtensionRegistryLite) registry);
  }
  public static final int MINIMUM_LIFETIME_FIELD_NUMBER = 10520;
  /**
   * <pre>
   * The minimum guaranteed duration for which the field value applies.
   *
   * For example, if an `idempotency_key` field has a lifetime of one hour, then
   * subsequent requests made after one hour should not assume any idempotency
   * guarantees from using the same idempotency key.
   *
   * This option only specifies a _minimum_ lifetime. APIs may honor fields for
   * longer than the minimum lifetime, and API consumers should not assume that
   * the minimum lifetime is a strict cutoff. For example, an API may honor an
   * idempotency key indefinitely, whatever the value of this field option.
   * </pre>
   *
   * <code>extend .google.protobuf.FieldOptions { ... }</code>
   */
  public static final
    com.google.protobuf.GeneratedMessage.GeneratedExtension<
      com.google.protobuf.DescriptorProtos.FieldOptions,
      com.google.protobuf.Duration> minimumLifetime = com.google.protobuf.GeneratedMessage
          .newFileScopedGeneratedExtension(
        com.google.protobuf.Duration.class,
        com.google.protobuf.Duration.getDefaultInstance());

  public static com.google.protobuf.Descriptors.FileDescriptor
      getDescriptor() {
    return descriptor;
  }
  private static  com.google.protobuf.Descriptors.FileDescriptor
      descriptor;
  static {
    java.lang.String[] descriptorData = {
      "\n\034aep/api/field_behavior.proto\022\007aep.api\032" +
      " google/protobuf/descriptor.proto\032\036googl" +
      "e/protobuf/duration.proto:d\n\020minimum_lif" +
      "etime\022\035.google.protobuf.FieldOptions\030\230R " +
      "\001(\0132\031.google.protobuf.DurationR\017minimumL" +
      "ifetimeB9\n\013dev.aep.apiB\022FieldBehaviorPro" +
      "toP\001Z\013aep.dev/api\370\001\001\242\002\003AEPb\006proto3"
    };
    descriptor = com.google.protobuf.Descriptors.FileDescriptor
      .internalBuildGeneratedFileFrom(descriptorData,
        new com.google.protobuf.Descriptors.FileDescriptor[] {
          com.google.protobuf.DescriptorProtos.getDescriptor(),
          com.google.protobuf.DurationProto.getDescriptor(),
        });
    minimumLifetime.internalInit(descriptor.getExtensions().get(0));
    com.google.protobuf.DescriptorProtos.getDescriptor();
    com.google.protobuf.DurationProto.getDescriptor();
  }

  // @@protoc_insertion_point(outer_class_scope)
}
```

Reply:
I wasn't worried about collisions per se - but the confusion here.

Just imagine that we'll have a sentence that says: "This field must be annotated with aep.field_behavior and google.api.field_behavior xyz".

But I guess the saving grace here is that for the AIPs the namespace is google.api and not AIP - so I guess it's fine. I still kind of lean toward calling this `aep.field` but it's a nit at this point.

Reply:
Right, I was actually _going for_ that correspondence between `google.api.field_behavior` and `aep.api.field_behavior`, because they are similar in spirit, and so it's just a matter of which things Google defined vs. which ones the AEPs define.

I can see how it might be confusing though, especially since the Google one is an enum and this one is a message.  Maybe `field_info` is actually a better correspondence?  Updated to that.